nucleus/modelci/__init__.py

```diff
@@ -72,7 +72,7 @@ def create_unit_test(
 
             unit_test = client.modelci.create_unit_test(
                 name="sample_unit_test",
-                slice_id="slc_bx86ea222a6g057x4380",
+                slice_id="YOUR_SLICE_ID",
```

Comment:
should it be lowercase to follow the format of others?

Reply:
oh I was going with the convention of `api_key="YOUR_SCALE_API_KEY"